### DISCUSSION TITLE
Request to add a source to research.

### DISCUSSION THREAD

**Discussion by jmtucny** [lululxvi/deepxde#1630] [Show and tell]
Dear Lu Lu,

We would like to add our recent publication:
https://www.sciencedirect.com/science/article/pii/S0045793023003390

Thank you again for your work. We are continuing to use DeepXDE in our research.

Sincerely yours,

Jean-Michel

**Comment by lululxvi** [comment]
Great. Feel free to submit a PR to update https://github.com/lululxvi/deepxde/blob/master/docs/user/research.rst
